Exdi/exdigdbsrvsample/GdbSrvControllerLib/TargetArchitectureHelpers.h

```diff
@@ -305,7 +349,9 @@ class TargetArchitectureHelpers
     }
 
     static void TokenizeAccessCode(_In_ const std::wstring& value, 
-        _In_z_ const wchar_t * delimiters, _Out_ std::vector<int>* pTokens)
+        _In_z_ const wchar_t* delimiters,
+        _In_z_ const wchar_t* format,

```

Comment:
Yes, Thanks, I'll add it, since scanf family function contains the annotation in stdio.h file
_Check_return_ _CRT_INSECURE_DEPRECATE_CORE(swscanf_s) _CRT_STDIO_IMP int __cdecl swscanf(_In_z_ const wchar_t * _Src, _In_z_ _Scanf_format_string_ const wchar_t * _Format, ...);

Reply:
but Scanf_format_string